components/checkbox/Checkbox.js

```diff
@@ -98,7 +98,11 @@ class Checkbox extends PureComponent {
           {...inputProps}
         />
         <span className={theme['shape']}>
-          <IconCheckmark className={theme['checkmark']} />
+          {partiallySelected ? (
```

Comment:
I would turn them around.
If you set both `partiallySelected` and `checked` to true, I would expect `checked` to get the upper hand, and show a checked checkbox.

Reply:
Damn good catch!

Reply:
Good catch, although I already made sure in the logic in the datagrid component that both checked  and partiallyChecked can not be true at the same time.

 :ok: by @Kemosabert on 06/07/2018 15:45:49